### PR TITLE
added validations for names, age and location

### PR DESCRIPTION
**Pull request by bdoughertys**
Added validations to user model

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -4,4 +4,8 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  validates :age, numericality: { only_integer: true, greater_than: 17, message: 'You must be older than 18' }
```
Comment (williamgrant04) [inline app/models/user.rb]: Given the `greater_than` the message should be "You must be 18 or older" or something along those lines

File: db/seeds.rb [hunk 2]
```diff
@@ -9,7 +9,7 @@
 #   end
 puts "Deleting all users..."
 
-User.destroy_all!
+User.destroy_all
```
Comment (williamgrant04) [inline db/seeds.rb:12]: I am stupid

**Review by williamgrant04 — CHANGES_REQUESTED** [review]
Fix below

File: app/models/user.rb (hunk 1 above)
Comment (bdoughertys) [reply app/models/user.rb]: Updated message to "You must be 18 or older"

**Review by bdoughertys — COMMENTED** [review]
Updated age restriction message

**Review by williamgrant04 — APPROVED** [review]
Approved

**Pull request merged**